additional_recipes/ros2-distro-mutex/recipe.yaml

```diff
@@ -35,6 +35,7 @@ requirements:
     - libprotobuf 5.29.3.*
     - libxml2 2.13.*
     - vtk 9.4.2.*
+    - graphviz 12.*
```

Comment:
Do you have a specific reason for having this here on top of the `conda_build_config.yaml`. Any dependency here just increases the risk of basic ROS packages (even the one that do not depend on graphviz at all) not be co-installable with conda-forge packages that depend on graphviz, and given that our rebuilds are not super-fast that is typically undesirable. Historically we added the constraints here to workaround bugs/missing data in package.xml metadata, but for graphviz I am not sure it is necessary,